packages/sdk/src/__tests__/judge-e2e.test.ts

```
@@ -0,0 +1,63 @@
+
+describe("judge mode e2e", () => {
+  it("loads input JSON and writes structured output JSON", async () => {
+    const root = await mkdtemp(join(tmpdir(), "obora-judge-"));
+    await mkdir(join(root, "artifacts"), { recursive: true });
+    await writeFile(
+      join(root, "artifacts", "submission.json"),
+      JSON.stringify({ title: "Sample", body: "Clear answer", rubric: { clarity: 1 } }, null, 2),
+      "utf8",
+    );
+
+    const workflow = await Workflow.fromYaml(join(process.cwd(), "src/__tests__/fixtures/judge-e2e.yaml"));
```

ESLint

[error] Replace `join(process.cwd(),·"src/__tests__/fixtures/judge-e2e.yaml")` with `⏎······join(process.cwd(),·"src/__tests__/fixtures/judge-e2e.yaml")⏎····`

(prettier/prettier)